flask/auth.py

```diff
@@ -134,7 +134,7 @@ def signup():
 
     if not registry_user:
         db.users.insert_one(user)
-
+        # send_verify_email(email)        #TODO: uncomment this line
```

Comment:
Are we verifying email during signup ? or when are we trying to verify the email. I saw the UI for verifying email in the code but was not able to see it when I tried signup.